front-end/.env

```diff
@@ -1 +1,2 @@
 REACT_APP_BACKEND_SERVICE=http://localhost:4000
+REACT_APP_CRYPTO_KEY=chungeek
```

Comment:
hahaha as much as I appreciate this let's change this into something more official.

I would suggest using any of the `Fort Knox Passwords` in https://randomkeygen.com/